### ISSUE TITLE
In autoupgrade module update nullable columns and fill them with default values

### ISSUE THREAD

**Issue by jolelievre** [PrestaShop/PrestaShop#34627]
One of the task of the null value problem (see parent Epic), we have some columns in the DB that should never use null values as it's not a use case expected or correctly handled, they should instead always be filled and have a default value in case no value is pecified

In the autoupgrade module all those columns should be fixed during the upgrade process, to maximize the nuùber of versions that would benefit from this the queries that update those fields should be put inside a function that can be reused

Then this function can be called in multiple upgrade versions scripts, all those queries have already been anticipated by @Hlavtox in this PR https://github.com/PrestaShop/autoupgrade/pull/605/files The queries themselves are good they should just be moved into a reusable function as recommended above

**Comment by M0rgan01** [comment]
It would also be necessary to change the default value of the columns concerned, to avoid the problem returning. There is a ticket for that https://github.com/PrestaShop/PrestaShop/issues/34628, but it only supports fresh installs in v9.

Also, there is the subject of null values, do we keep the column nullable? this may be redundant with the default value, but helps maintain data consistency

**Comment by M0rgan01** [comment]
EDIT: Adding a default value to a lot of columns does not work 

`Syntax error or access violation: 1101 BLOB, TEXT, GEOMETRY or JSON column '...' can't have a default value`


It seems that this is not possible in v5 https://dev.mysql.com/doc/refman/5.7/en/data-type-defaults.html

> The [BLOB](https://dev.mysql.com/doc/refman/5.7/en/blob.html), [TEXT](https://dev.mysql.com/doc/refman/5.7/en/blob.html), GEOMETRY, and [JSON](https://dev.mysql.com/doc/refman/5.7/en/json.html) data types cannot be assigned a default value.

But is possible in v8 with a function call https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html

**Comment by Hlavtox** [comment]
@M0rgan01 Yes maybe the default value is not possible for these columns - https://docs.google.com/spreadsheets/d/1DYQ4qdWN0YWSCl7RuibMn6aEdXte9Fd7RYz3iJ10hWA/edit?usp=sharing :-)

**Comment by Hlavtox** [comment]
@WahbiPS Not completed

**Comment by gericfo** [comment]
Related to https://github.com/PrestaShop/autoupgrade/pull/656

**Comment by gericfo** [comment]
Thanks @Hlavtox for your feedback. Crystal clear!
